go.mod

```diff
@@ -3,8 +3,9 @@ module github.com/vercel/terraform-provider-vercel
 go 1.21
 
 require (
-	github.com/hashicorp/terraform-plugin-framework v1.7.0
-	github.com/hashicorp/terraform-plugin-go v0.22.1
+	github.com/hashicorp/terraform-plugin-framework v1.10.0
+	github.com/hashicorp/terraform-plugin-framework-validators v0.13.0
+	github.com/hashicorp/terraform-plugin-go v0.23.0
```

Comment:
I usually try to do package bumps as a separate PR, but not worth separating out now.